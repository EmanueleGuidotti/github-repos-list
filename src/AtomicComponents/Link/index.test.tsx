import React from "react";
import renderer from "react-test-renderer";
import LinkComponent from "./index";

describe("LinkComponent", () => {
  it("Should render link component", () => {
    const component = renderer.create(
      <LinkComponent link="https://github/test-link/" text="Some github repo" />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
