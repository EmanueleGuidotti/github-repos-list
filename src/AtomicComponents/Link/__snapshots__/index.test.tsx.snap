// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LinkComponent Should render link component 1`] = `
<a
  className="ant-typography"
  href="https://github/test-link/"
  rel="noopener noreferrer"
  style={
    Object {
      "WebkitLineClamp": undefined,
      "textDecoration": "underline",
    }
  }
  target="_blank"
>
  Some github repo
</a>
`;
